<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <arg name="des_x" default="-1" />
    <arg name="des_y" default="1" />
    <param name="des_pos_x" value="$(arg des_x)" />
    <param name="des_pos_y" value="$(arg des_y)" />
    <node pkg="project_3" type="follow_wall.py" name="wall_follower" />
    <node pkg="project_3" type="go_to_point.py" name="go_to_point" />
    <node pkg="project_3" type="bug1.py" name="bug1" output="screen" />
</launch>
